.github/workflows/helm-test.yml

```
@@ -105,3 +111,14 @@ jobs:
                     helm repo add postgresql https://charts.bitnami.com/bitnami
                     ct install --charts charts/puris --helm-extra-set-args "--set backend.image.repository=${{ env.REGISTRY }}/${{ env.APP_BACKEND_NAME}} --set backend.image.tag=${{ env.TAG }} --set frontend.image.repository=${{ env.REGISTRY }}/${{ env.APP_FRONTEND_NAME }} --set frontend.image.tag=${{ env.TAG }}"
                 if: github.event_name != 'pull_request' || steps.list-changed.outputs.changed == 'true'
+
+            # Upgrade the released chart version with the locally available chart
+            # default value for event_name != workflow_dispatch
+            -   name: Run helm upgrade
+                run: |
+                    helm repo add bitnami https://charts.bitnami.com/bitnami
+                    helm repo add postgresql https://charts.bitnami.com/bitnami
+                    helm install puris charts/puris --version ${{ github.event.inputs.upgrade_from || '1.0.0' }}
+                    helm dependency update charts/puris
+                    helm upgrade [NAME] charts/puris
```

Replace [NAME] placeholder. 

```suggestion
                    helm upgrade puris charts/puris
```